tests/unit/test_is_stuck.py

```diff
@@ -29,14 +28,17 @@ def collect_events(stream):
 
 
 @pytest.fixture
-def event_stream():
-    with tempfile.TemporaryDirectory() as temp_dir:
-        file_store = get_file_store('local', temp_dir)
-        event_stream = EventStream('asdf', file_store)
-        yield event_stream
-
-        # clear after each test
-        event_stream.clear()
+def temp_dir(tmp_path_factory):
```

Comment:
It'd be nice to add typing here, I wasn't sure what class this is. Ditto below.

Reply:
done!